src/python/doc/rips_complex_user.rst

```diff
@@ -378,7 +378,7 @@ Example from a point cloud combined with DistanceToMeasure
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Combining with DistanceToMeasure, one can compute the DTM-filtration of a point set, as in `this notebook <https://github.com/GUDHI/TDA-tutorial/blob/master/Tuto-GUDHI-DTM-filtrations.ipynb>`_. 
-Remark that `DTMRipsComplex <rips_complex_user.html#dtm-rips-complex>`_ class provides exactly this function.
+Remark that `DTM Rips Complex (user manual) <rips_complex_user.html#dtm-rips-complex>`_ class provides exactly this function.
```

Comment:
It is a bit confusing to say that it is a class, but put spaces in the middle of its name. Bah...

Reply:
That's fair : 83db295e